ocfweb/docs/views/servers.py

```diff
@@ -57,81 +56,90 @@ def has_munin(self):
         return self.type in ('hypervisor', 'vm', 'server', 'desktop')
 
 
+def is_hidden(host):
+    return host['cn'][0].startswith('hozer-') or host['cn'][0].startswith('dev-')
+
+
+TEMP_GET_CHILDREN = "facts { name = 'vms' }"
```

Comment:
There's no need to name these constants `TEMP`. (After all, they're not temporary.) Rather, I would suggest a name like `PQL_GET_CHILDREN`.